build-tests/recorder.mdx

```diff
@@ -59,11 +59,11 @@ The modals can be dragged by the six-dot icon and moved for better visibility.
 
 Where can you use the Recorder?
 
-You can flexibly hand over between the Recorder and Agent during the **test generation process**, in the **editing** and the **repair mode** to help you re-record failed steps. You can also record a **new test case from scratch**.
+You can flexibly hand over between the Recorder and Agent during the **test generation process**, in the **editing** and the **repair mode** to help you re-record failed steps. You can record a **new test case from scratch**.
 
 ### 1. Take over from the Agent during test generation
 
-When the Agent generates a test case it might take a path you don't want it to. You can stop it and take over from it. It opens the Recorder at the spot the Agent left off and you can proceed recording the next steps of the test case.
+When the Agent generates a test case it might take a path you dont want it to . You can stop it and take over from it . It open the Recorder at the spot the Agent left off and you can proceed recording next steps the test case.
```

Comment:
```suggestion
When the Agent generates a test case it might take a path you don't want it to. You can stop it and take over from it. It opens the Recorder at the spot the Agent left off and you can proceed recording the next steps of the test case.
```